src/providers/structure.ts

```diff
@@ -4,6 +4,7 @@ import {latexParser,bibtexParser} from 'latex-utensils'
 
 import type { Extension } from '../main'
 import * as utils from '../utils/utils'
+import { PathRegExp, MatchPath, MatchType } from '../components/managerlib/pathutils'
```

Comment:
Please stop importing `managerlib/pathutils` directly here, which violates the hierarchy of libraries. It would be better to consider moving `managerlib/pathutils` to `components/` and expose it as a component.

Reply:
That's exactly what I thought when I import that lib. Quite strange why it appears there.

Reply:
After looking into the `pathutils` code, I think it would be more appropriate to refactor that class in another dedicated PR. I will work on it immediately after this PR merged.